test/Feature/AddressOfLabels.c

```diff
@@ -0,0 +1,36 @@
+// RUN: %llvmgcc -emit-llvm -g -c %s -o %t.bc
+// RUN: rm -rf %t.klee-out
+// RUN: not %klee --output-dir=%t.klee-out %t.bc 2> %t.log
+// RUN: FileCheck --input-file %t.log %s
+
+/* 
+   This currently tests that KLEE clearly reports that it doesn't
+   support taking the address of the labels, in particular
+   blockaddress constants.
+
+   The test would need to be changes once support for this feature is
+   added. 
+*/
+
+#include <stdio.h>
+
+int main (int argc, char** argv)
+{
+  int i = 1;
+  // CHECK: Cannot handle constant 'i8* blockaddress 
+  // CHECK: AddressOfLabels.c:[[@LINE+1]]
```

Comment:
Huh. I just a test and indeed `FileCheck` seems to be happy with this. I have always thought a sequence of `CHECK:`s would match different lines. Learnt something new there.

If were using modern FileCheck the second `CHECK` would be a `CHECK-SAME:` but it's not available in LLVM 3.4 http://releases.llvm.org/3.4/docs/CommandGuide/FileCheck.html